pcsx2/COP0.cpp

```diff
@@ -45,6 +45,13 @@ void __fastcall WriteCP0Status(u32 value) {
     cpuSetNextEventDelta(4);
 }
 
+void __fastcall WriteCP0Config(u32 value) {
+	// Protect the read-only ICacheSize (IC) and DataCacheSize (DC) bits
+	cpuRegs.CP0.n.Config = value & ~0xFC0;
+	cpuRegs.CP0.n.Config |= 0x440;
+	cpuUpdateOperationMode();
+	cpuSetNextEventDelta(4);
```

Comment:
I don't think it needs these 2 last lines. the top one of the two doesn't do anything anyway.

Reply:
Should I purge those two calls for the status register as well or leave them be? Just wanted to be consistent.

Reply:
the cpuSetNextEventDelta needs to be there for the status register because of interrupts, but you don't need it for the Config register.

Reply:
the cpuUpdateOperationsMode thing can probably just go.. lol